bokeh/tests/test_properties.py

```diff
@@ -317,14 +317,28 @@ class IsDelegate(HasProps):
 
         class IncludesDelegateWithPrefix(HasProps):
             z = Include(IsDelegate, use_prefix=True)
-            z_y = String("world") # override the Include
+            z_y = Int(57) # override the Include
 
         class IncludesDelegateWithoutPrefix(HasProps):
             z = Include(IsDelegate, use_prefix=False)
-            y = String("world") # override the Include
+            y = Int(42) # override the Include
+
+        class IncludesDelegateWithoutPrefixUsingOverride(HasProps):
+            z = Include(IsDelegate, use_prefix=False)
+            y = Override(default="world") # override the Include changing just the default
```

Comment:
All I meant is that "conceptually" the effect of Include is similar to deriving from https://github.com/bokeh/bokeh/blob/master/bokeh/mixins.py - for example, rather than Include, we could have a function that took a class and prefixed all its properties, and then we could inherit from that class:

```
LabelTextProps = prefix_properties_in_class(TextProps, 'label')
class Axis(LabelTextProps):
  ...
```

So that's why it could "make sense" that Override can override Include'd props.

I don't think there's much advantage to _actually_ doing it the above way with the autogenerated class, `Include` is fine I think or even a little bit tidier because it doesn't put weird stuff in the class hierarchy.